### ISSUE TITLE
[typo] url is inaccessible

### ISSUE THREAD

**Issue by Freed-Wu** [tesseract-ocr/tessdoc#55]
<https://github.com/tesseract-ocr/tessdoc/blob/master/ImproveQuality.md>

```
If you know you will only encounter a subset of the characters available in the language, such as only digits, you can use the `tessedit_char_whitelist` [configuration variable](ControlParams). See the [FAQ for an example](FAQ-Old#how-do-i-recognize-only-digits).
```

`ControlParams` cannot be accessed.

**Comment by stweil** [comment]
The link moved to a different location: https://github.com/tesseract-ocr/tessdoc/blob/master/tess3/ControlParams.md.

We need a fix for ImproveQuality.md.

**Comment by kseniazhagorina** [comment]
In your documentation you have also some incorrect links
wget https://github.com/tesseract-ocr/tessdata/raw/master/eng.traineddata
instead
wget https://github.com/tesseract-ocr/tessdata/raw/main/eng.traineddata

**Comment by stweil** [comment]
Yes, we recently renamed the branch from `master` to `main`, and some parts of the documentation still need to be updated.
